common/djangoapps/enrollment/management/commands/enroll_user_in_course.py

```diff
@@ -0,0 +1,56 @@
+"""
+Management command for enrolling a user into a course via the enrollment api
+"""
+from optparse import make_option
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from enrollment.api import add_enrollment
+
+
+class Command(BaseCommand):
+    """
+    Enroll a user into a course
+    """
+    help = """
+    This enrolls a user into a given course as "audit", "verified" or "honor". Email is required.
+
+    example:
+        # Enroll a user test@example.com into the demo course
+        manage.py ... enroll_user_in_course -e test@example.com -c edX/Open_DemoX/edx_demo_course -m verified
+
+        If a user is already enrolled, the command will return an error.
+    """
+
+    option_list = BaseCommand.option_list + (
+        make_option('-m', '--mode',
+                    metavar='ENROLLMENT_MODE',
+                    dest='mode',
+                    default='honor',
+                    choices=('audit', 'verified', 'honor'),
+                    help='Enrollment type for user for a specific course'),
+        make_option('-e', '--email',
+                    metavar='EMAIL',
+                    dest='email',
+                    default=None,
+                    help='Email for user'),
+        make_option('-c', '--course',
+                    metavar='COURSE_ID',
+                    dest='course',
+                    default=None,
+                    help='course ID to enroll the user in'),
+    )
+
+    def handle(self, *args, **options):
+        """
+        Get and enroll a user in the given course. Mode is optional and defers to the enrollment API for defaults.
+        """
+        email = options['email']
+        course = options['course']
+        mode = options['mode']
+
+        if email is None or course is None:
+            raise Exception('Need course ID and user e-mail.')
```

Comment:
This phrasing might be difficult for a non-native english speaker to understand. Maybe change it to something like "Course ID and user email address are required."